Doc/library/signal.rst

```diff
@@ -237,6 +237,12 @@ The :mod:`signal` module defines the following functions:
    :func:`sigpending`.
 
 
+.. function:: raise_signal(signum)
+
+   Sends a signal to the executing process. Returns nothing. If a signal handler
+   is called, the *raise* function shall not return until after the signal handler does.
```

Comment:
>  Sends a signal to the executing process. 

IMO "to the calling process" reads better.

> If a signal handler
> is called, the *raise* function shall not return until after the signal handler does.

While this is true for the C `raise` function, it's not for Python.  In Python, signals are queued and their handlers are executed at some undetermined point later.  I suggest removing this sentence.